src/components/tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -5,16 +5,21 @@ import sinon from 'sinon';
 import { shallow } from 'enzyme';
 import noop from 'lodash/noop';
 
-import Tooltip, { TooltipPosition, TooltipTheme } from '../Tooltip';
+import { TooltipBase as Tooltip, TooltipPosition, TooltipTheme } from '../Tooltip';
 import TetherPosition from '../../../common/tether-positions';
 
 const sandbox = sinon.sandbox.create();
 
 describe('components/tooltip/Tooltip', () => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const mockIntl: any = {
```

Comment:
could we move this outside and above the `describe`? in case we ever add another `describe` it could be used for both